### PR TITLE
Added template for vector of integers

### PR DESCRIPTION
**Pull request by jessemapel**
This also fixes building with Python 3.8 and Anaconda

### CODE REVIEW

File: python/FindPythonAnaconda.cmake [hunk 1]
```diff
@@ -61,15 +61,16 @@ if(NOT CMAKE_FIND_ANACONDA_PYTHON_INCLUDED)
     set( _py_version_patch ${CMAKE_MATCH_3} )
     set( ANACONDA_PYTHON_VERSION ${_py_version_major}.${_py_version_minor} )
 
-    
-    if( ${_py_version_major} MATCHES 2 )
+
+    if( (${_py_version_major} MATCHES 2) OR
+        (${_py_version_major} MATCHES 3 AND ${_py_version_minor} MATCHES 8) )
```
Comment (ladoramkershner) [inline python/FindPythonAnaconda.cmake]: Is this a change moving forward? If so should this be major >= 3 and minor >= 8?

Comment (jessemapel) [reply python/FindPythonAnaconda.cmake]: I'm not sure I can look into this

Comment (jessemapel) [reply python/FindPythonAnaconda.cmake]: This is dependent upon how the Python distribution was compiled. For 3.8, Anaconda compiled it without pymalloc, hence no m, but we don't know how they will compile 3.9 so we'll have to deal with that when 3.9 becomes the standard for Anaconda.

**Comment by jessemapel** [comment]
Closing and re-opening to redo Travis build that failed with https error

File: python/FindPythonAnaconda.cmake (hunk 1 above)
Comment (ladoramkershner) [reply python/FindPythonAnaconda.cmake]: Will this error in a not silent way if _py_ext is incorrect?

Comment (jessemapel) [reply python/FindPythonAnaconda.cmake]: no, it will fail to link while building

**Pull request merged**